### DISCUSSION TITLE
Why `catalog` in `getTables` of `DuckDBDatabaseMetaData` not supported?

### DISCUSSION THREAD

**Discussion by xxks-kkk** [duckdb/duckdb#6473] [Q&A] [answered]
I'm using the latest JDBC driver (0.7.0) and try to get table I just created using the `getTables`. I found out that `catalog` argument is not supported shown below. I was wondering why? I noticed that for in-memory, the catalog from `connection.getCatalog()` is `memory` and database filename for persist database. I used this information to create table and can do so successfully. But, why can I use this same information to get table I just created?

Thank you!

```
	public ResultSet getTables(String catalog, String schemaPattern, String tableNamePattern, String[] types)
			throws SQLException {
		if (catalog != null && !catalog.isEmpty()) {
			throw new SQLException("Actual catalog argument is not supported, got " + catalog);
		}
```

**Comment by Mause** [comment, marked as answer]
Can you please try with the latest dev build? This should already be fixed
